src/app/plugins/platform/Docker/spawnComposer.ts

```diff
@@ -1,50 +1,51 @@
+import { ChildProcess, spawn, SpawnOptions } from 'child_process';
 import envPaths from 'env-paths';
-import execa, { ExecaReturns, Options } from 'execa';
```

Comment:
Swapped to child_process because that's what was built into Yeoman natively which made it a lot more portable to pass around between contexts. My preference was to remove additional dependencies unless they were needed.

Otherwise, yes I did run into consistent issues with execa properly executing within Docker. I never fully identified what the cause of it was, but the built-in Yeoman functions worked more readily with no identified draw-backs.